src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
+@import 'card';
+@import 'stars';
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
```

Comment:
Importing Google Fonts directly in an SCSS file is not standard practice. Consider moving this import to the HTML file within a `<link>` tag in the `<head>` section for better performance and maintainability.